bingo/bingo-elastic/python/bingo_elastic/elastic.py

```diff
@@ -315,14 +317,16 @@ def filter(
         self,
         similarity: Union[BaseMatch] = None,
         exact: IndigoRecord = None,
-        substructure: IndigoRecord = None,
-        limit=10,
+        substructure: IndigoObject = None,
+        indigo_session: Indigo = None,
+        limit: int = 5000,
```

Comment:
Why 5000?

Reply:
@AATDev21 @mkviatkovskii limit of 5000 would be substantial for Elasticsearch. We should stay with reasonable defaults. I’m okay with 20 but not 5000

Reply:
Explained giant limit below inside the similar comment and set it back to 10!